src/print-prime-numbers-from-1-to-1000.js

```diff
@@ -0,0 +1,17 @@
+function isPrime(n) {
```

Comment:
remove this, using existing function from [check-primes.js](https://github.com/anhdiepmmk/algorithm/blob/master/src/check-primes.js) service

Reply:
const { isPrimeNumber } = require('./check-primes ');